core/src/main/scala/io/chrisdavenport/shellfish/Shell.scala

```diff
@@ -285,58 +285,57 @@ object Shell {
     def mktree(path: String): F[Unit] =
       getResolved(path).flatMap(files.createDirectories(_).void)
 
-    def rm(path: String): F[Unit] = getResolved(path).flatMap(files.delete(_))
+    def rm(path: String): F[Unit] = getResolved(path).flatMap(files.delete)
     def rmDir(path: String): F[Unit] =
-      getResolved(path).flatMap(files.deleteDirectoryRecursively(_))
+      getResolved(path).flatMap(files.deleteRecursively)
 
     def symlink(createAt: String, linkTo: String): F[Unit] =
       for {
         r1 <- getResolved(createAt)
         r2 <- getResolved(linkTo)
-        _  <- Sync[F].delay(Files.createSymbolicLink(r1, r1))
+        _  <- files.createSymbolicLink(r1, r2)
       } yield ()
 
-    def isNotSymLink(path: String): F[Boolean] = getResolved(path).flatMap(p =>
-      Sync[F].delay {
-        !Files.isSymbolicLink(p)
-      }
-    )
+    def isNotSymLink(path: String): F[Boolean] = for {
+      newPath <- getResolved(path)
+      symlink <- files.isSymbolicLink(newPath)
+    } yield !symlink
 
     def testFile(path: String): F[Boolean] =
-      getResolved(path).flatMap(files.isFile(_))
+      getResolved(path).flatMap(files.isRegularFile)
     def testDir(path: String): F[Boolean] =
-      getResolved(path).flatMap(files.isDirectory(_))
+      getResolved(path).flatMap(files.isDirectory)
     def testPath(path: String): F[Boolean] =
-      getResolved(path).flatMap(files.exists(_))
+      getResolved(path).flatMap(files.exists)
 
     def date: F[Instant] =
       cats.effect.Clock[F].realTime.map(_.toMillis).map(Instant.ofEpochMilli)
     // lastModified
     def dateFile(path: String): F[Instant] = for {
       p <- getResolved(path)
-      i <- Sync[F].delay(Files.getLastModifiedTime(p).toInstant)
-    } yield i
+      t <- files.getLastModifiedTime(p)
+    } yield Instant.ofEpochMilli(t.toMillis)
 
     def touch(path: String): F[Unit] = for {
       p   <- getResolved(path)
-      now <- date
+      now <- cats.effect.Clock[F].realTime
       _ <- exists(p.toString).ifM(
-        Sync[F]
-          .delay(
-            Files.setLastModifiedTime(
-              p,
-              java.nio.file.attribute.FileTime.from(now)
-            )
+        files
+          .setFileTimes(
+            path = p,
+            lastModified = Some(now),
+            lastAccess = None,
+            creationTime = Some(now),
```

Comment:
```suggestion
            creationTime = None,
```
I believe We should use `None` here, as `touch` only modifies last modified time.